internal/services/automation/automation_dsc_configuration_resource_test.go

```diff
@@ -91,7 +91,7 @@ resource "azurerm_automation_dsc_configuration" "test" {
   location                = azurerm_resource_group.test.location
   content_embedded        = "configuration acctest {}"
   description             = "test"
-
+  log_verbose             = "true"
```

Comment:
this is an optional property? it should not be on the basic test, please add it to the complete test